src/src/components/ObjectBrowser.jsx

```diff
@@ -3031,6 +3033,8 @@ class ObjectBrowser extends Component {
                 } else {
                     delete this.objects[id];
                 }
+            } else if (obj) {
```

Comment:
would follow the same recommendation as above

Reply:
as stated above